### PR TITLE
adds more guns to summon guns

### PR DESCRIPTION
**Pull request by SonixApache**
I figure the ballistic pistols aren't that big a deal, but maybe the energy weapons are? Discuss.

🆑 
- tweak: summon guns now summons a larger variety of guns

### CODE REVIEW

**Comment by Blithering** [comment]
considering the gyrojet and pulse rifle are already on the list, there's not much you can add that would tip the balance. More variety is good

**Comment by jknpj** [comment]
fuck it

File: code/game/gamemodes/wizard/rightandwrong.dm [hunk 1]
```diff
@@ -28,18 +28,26 @@
...
+		var/randomizeguns = pick("taser","stunrevolver","egun","laser","retro","laserak","revolver","detective","smg","nuclear","deagle","gyrojet","pulse","silenced","cannon","doublebarrel","shotgun","combatshotgun","mateba","smg","uzi","crossbow","saw","hecate","osipr","gatling","bison","ricochet","spur","nagant","obrez","beegun","beretta","usp","glock","luger","colt","plasmapistol")
 		var/randomizemagic = pick("fireball","smoke","blind","mindswap","forcewall","knock","horsemask","blink","disorient","staffchange","armor","scrying", "clowncurse", "mimecurse", "shoesnatch", "robesummon")
 		var/randomizeswords = pick("unlucky", "misc", "glass", "throw", "armblade", "pickaxe", "pcutter", "esword", "alt-esword", "machete", "kitchen", "spear", "katana", "axe", "venom", "boot", "saw", "scalpel", "bottle", "switchtool")
 		var/randomizeknightcolor = pick("green", "yellow", "blue", "red", "templar")
 		if(summon_type == "guns")
 			switch (randomizeguns)
 				if("taser")
 					new /obj/item/weapon/gun/energy/taser(get_turf(H))
+				if("taser")
```
Comment (DamianX) [inline code/game/gamemodes/wizard/rightandwrong.dm]: this should be `"stunrevolver"`

Comment (SonixApache) [reply code/game/gamemodes/wizard/rightandwrong.dm]: goddamnit

**Review by DamianX — APPROVED** [review]
Am I reading that right? The beretta was in the pick() list but didn't have a corresponding if()?

**Comment by Exxion** [comment]
Can anyone tell me why it isn't just a list of types?

**Comment by DamianX** [comment]
No reason. It should be a list of lists though, since some weapons also need to spawn the ammo.

**Comment by Exxion** [comment]
Fair enough
I'd recommend making it an associative list, where the keys are the primary spawn and the values are the auxiliary spawns (such as ammo), if any
So spawn the type in the list, then if there is a value associated with that type, spawn everything in that value as well

BYOND lists being simultaneously associative and normal lists sucks but it lets you do shit like that

**Comment by SonixApache** [comment]
@DamianX yes you are reading that right

@Exxion those are a lot of big words you're writing there

**Comment by Exxion** [comment]
It's like three of them

**Comment by DamianX** [comment]
Are you gonna do the thingy or should I merge this as-is?

**Comment by SonixApache** [comment]
i'm not gonna do the thingy

**Pull request merged**